### PR TITLE
Ewb 586 support writing database

### PR DESCRIPTION
**Pull request by kgreav**

### CODE REVIEW

**Comment by ryanjcoleman** [comment]
Task linked: [EWB-586 Support writing database](https://app.clickup.com/t/6929263/EWB-586)

File: src/zepben/evolve/database/sqlite/writers/base_cim_writer.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    Helper methods for building insert statements. Note all fields need to be populated.
+    """
+
+    database_tables: DatabaseTables
+    cursor: Cursor
+    _failed_ids: Set[str] = set()
+
+    def _save_document(self, table: TableDocuments, insert: PreparedStatement, document: Document, description: str) -> bool:
+        insert.add_value(table.title.query_index, document.title)
+        # TODO: JVM seems to use Z as TZ offset (for UTC+0?) while python uses +HH:mm format. Need to investigate here
```
Comment (kgreav) [inline src/zepben/evolve/database/sqlite/writers/base_cim_writer.py:42]: can remove this TODO. also we're getting test failures in the CI I believe from here. Possibly related to test data using the wrong format? check the test errors I guess (or try run them locally)

**Review by kgreav — COMMENTED** [review]
get the tests working.

**Comment by kgreav** [comment]
Merged in #71

**Pull request closed without merging**